src/_posts/2018-08-XX-review-input-streaming-connectors.md

```diff
@@ -41,15 +41,15 @@ Below are the main streaming input connectors for available for Beam and Spark D
    </td>
    <td>Local<br>(Using the <code>file://</code> URI)
    </td>
-   <td><a href="https://beam.apache.org/documentation/sdks/javadoc/2.5.0/org/apache/beam/sdk/io/TextIO.html">TextIO</a>
+   <td><a href="https://beam.apache.org/documentation/sdks/javadoc/2.6.0/org/apache/beam/sdk/io/TextIO.html">TextIO</a>
```

Comment:
I am not sure if this will work but it would be good to refer to the urls by using the latest URLs (this work in normal markdown links so hopefully it will work here too:
`{{ site.baseurl }}/documentation/sdks/javadoc/{{ site.release_latest }}/org/apache/beam/sdk/io/BoundedSource.html`